uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/calender-instance_test.js

```diff
@@ -0,0 +1,91 @@
+define([
+  'mocks/data/work-pattern-data',
+  'leave-absences/shared/models/instances/calender-instance',
+], function (mockData) {
+  'use strict';
+
+  describe('CalenderInstance', function () {
+    var CalenderInstance;
+
+    beforeEach(module('leave-absences.models.instances'));
+
+    beforeEach(inject([
+      "CalenderInstance",
+      function (_CalenderInstance_) {
+        CalenderInstance = _CalenderInstance_;
+        CalenderInstance.calenderData = mockData.calenderData().values;
+      }]
+    ));
+
+    describe('isWorkingDay()', function () {
```

Comment:
Is it necessary to move each test case to a new `describe` block? We are testing a single test case there.